animations/lib/src/misc/expand_card.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+
+class ExpandCardDemo extends StatelessWidget {
+  static String routeName = '/expand_card';
+
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(),
+      body: Center(
+        child: ExpandCard(),
+      ),
+    );
+  }
+}
+
+class ExpandCard extends StatefulWidget {
+  _ExpandCardState createState() => _ExpandCardState();
+}
+
+class _ExpandCardState extends State<ExpandCard>
+    with SingleTickerProviderStateMixin {
+  static const Duration duration = Duration(milliseconds: 300);
+  bool expanded = false;
+
+  double get size => expanded ? 200 : 100;
+
+  void initState() {
+    super.initState();
+  }
+
+  void toggleExpanded() {
+    setState(() {
+      expanded = !expanded;
+    });
+  }
+
+  Widget build(context) {
+    return GestureDetector(
+      onTap: () => toggleExpanded(),
+      child: Card(
+        child: AnimatedCrossFade(
+          alignment: Alignment.center,
+          duration: duration,
+          crossFadeState:
+              expanded ? CrossFadeState.showSecond : CrossFadeState.showFirst,
+          firstChild: AnimatedContainer(
```

Comment:
Could (and I'm really asking here, I haven't checked) the AnimatedContainer be moved up the tree to be the parent of AnimatedCrossFade (which could then have the Images as its direct descendants? If it's just there to control the size of the card, you might not need two of them this low.